### PR TITLE
Create the docs of the app

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request? What problem is this solving?

Add the docs of the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,240 @@
...
+        <Slide key={product.id}>
+          <DisplayProductComponent product={product} />
+        </Slide>
+      ))}
+    </Slider>
+  )
+}
+```
+
+And all the components together:
```
Comment (theomoura) [inline README.md]: Bellow is an example with the all components together:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,240 @@
...
+| `cursor` | `string` | :no_entry_sign: | `'-webkit-grab'` | Css value of cursor when mouse is hovering the slider frame |
+| `cursorOnMouseDown` | `string` | :no_entry_sign: | `'-webkit-grabbing'` | Css value of cursor when mouse is down |
+| `duration` | `number` | :no_entry_sign: | `250` | Duration of transitions |
+| `easing` | `string` | :no_entry_sign: | `'ease-out'` | Transition function |
+| `loop` | `bool` | :no_entry_sign: | `false` | If the slides should be looping |
+| `onChangeSlide` | `func` | :heavy_check_mark: | :no_entry_sign: | Function to change the value of currentSlide. The function should expect a `number` as it's only parameter |
+| `perPage` | `number/object` | :no_entry_sign: | `1` | Amount of slides to be on the screen, if a number is passed, then that's the number of slides that will be shown, if an objecdt with breakpoints is passed, then the component will check the size of the screen to see how many slides will be on the screen at the same time |
```
Comment (theomoura) [inline README.md]:
```suggestion
| `perPage` | `number/object` | :no_entry_sign: | `1` | Amount of slides to be on the screen, if a number is passed, then that's the number of slides that will be shown, if an object with breakpoints is passed, then the component will check the size of the screen to see how many slides will be on the screen at the same time |
```

**Review by theomoura — CHANGES_REQUESTED** [review]
In the configuration section, check if there is enum type prop and relate it possible values

**Review by theomoura — APPROVED** [review]

**Review by ArthurSampaio — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,240 @@
...
+| `totalSlides` | `number` | :heavy_check_mark: | :no_entry_sign: | Total value of sliders that will be rendered |
+| `showDotsPerPage` | `bool` | :no_entry_sign: | `false` | If this frag is true, then every dot represent a page of slides (e.g. if `perPage = 2` and you have 4 elements, then you have 2 dots), if false, then it will render one dot to each slide |
+
+## Styles API
+You can style this app by using the props `classeName` and `classes` of the components. But if you want to style every slider of your app you need to use the [CSS namespaces](#css-namespaces) to do it.
+
+### CSS namespaces
+:construction: :construction: :construction:
+
+## Troubleshooting
```
Comment (klzns) [inline README.md]: I would remove this section.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,240 @@
...
+
+If you want to show multiple items in the same page at the same time you can use the prop `perPage`:
+
+```javascript
+render() {
+  const { currentSlide } = this.state
+  const { myProducts } = this.props
+
+  const perPage = {
+    1300: 5,
```
Comment (klzns) [inline README.md:75]: Explain what 1300 is

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by klzns — APPROVED** [review]

**Pull request merged**